http://hg.punctweb.ro

--- a/dwm.c	2012-04-15 15:03:17.662134727 +0300
+++ b/dwm.c	2012-04-15 15:03:17.665134727 +0300
@@ -847,6 +847,8 @@ drawtext(const char *text, unsigned long
 
 void
 enternotify(XEvent *e) {
+	if(clicktofocus) return;
+
 	Client *c;
 	Monitor *m;
 	XCrossingEvent *ev = &e->xcrossing;
@@ -1238,6 +1240,8 @@ monocle(Monitor *m) {
 
 void
 motionnotify(XEvent *e) {
+	if(clicktofocus) return;
+
 	static Monitor *mon = NULL;
 	Monitor *m;
 	XMotionEvent *ev = &e->xmotion;
